### PR TITLE
Add configurable current_user method

### PR DESCRIPTION
**Pull request by chaadow**
There are also other cool and undocumented configs ( such as on_job_update ) 

I don't where to put them in the README though.

### CODE REVIEW

**Comment by ankane** [comment]
Hi @chaadow, thanks for another PR. The code looks good, but let's call it `user_method` instead of `current_user_method`.

**Comment by chaadow** [comment]
@ankane thanks for the suggestion, I've applied and amended it.

File: lib/clockwork_web.rb [hunk 1]
```diff
@@ -18,9 +18,11 @@ class << self
     attr_accessor :monitor
     attr_accessor :running_threshold
     attr_accessor :on_job_update
+    attr_accessor :current_user_method
```
Comment (ankane) [inline lib/clockwork_web.rb]: This needs updated as well.

Comment (chaadow) [reply lib/clockwork_web.rb]: Sorry, it's done thanks!

**Comment by ankane** [comment]
Great, thanks!

**Pull request merged**